src/components/FeatureSelector/Defaults.tsx

```diff
@@ -26,7 +28,8 @@ const options: FeatureSliderData[] = [
     max: 100,
     currentMin: 0,
     currentMax: 100,
-    units: '%'
+    units: '%',
+    description : 'Energy represents a perceptual measure of intensity and activity.'
```

Comment:
```suggestion
    description : 'Energy describes intensity and activity of a track.'
```
wtf is a perceptual measure??